terraform/workspace-variables/dev.tfvars

```diff
@@ -0,0 +1,14 @@
+# Platform
+environment = "dev"
+app_environment = "dev"
+parameter_store_environment = "dev"
+
+# Gov.UK PaaS
+paas_api_url = "https://api.london.cloud.service.gov.uk"
+paas_space_name = "sandbox"
+paas_postgres_service_plan = "tiny-unencrypted-11"
+paas_app_start_timeout = "180"
+paas_app_stopped = false
+paas_web_app_deployment_strategy = "blue-green-v2"
+paas_web_app_instances = 1
+paas_web_app_memory = 256
```

Comment:
Might need to up this to 512 for dev

Reply:
And how/when will we know? When we see some app crashes?

Reply:
Also, if we want to add another environment, do we need to add another one of these files?

Reply:
Yes and yes